<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="SNOW BDD Test Suite" duration-ms="73017" started-at="2019-08-04T20:17:12Z" finished-at="2019-08-04T20:18:25Z">
    <groups>
      <group name="cucumber">
        <method signature="TestRunner.feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:com.snowTest.testRunner.TestRunner@4883b407]" name="feature" class="com.snowTest.testRunner.TestRunner"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Regression Test" duration-ms="73017" started-at="2019-08-04T20:17:12Z" finished-at="2019-08-04T20:18:25Z">
      <class name="com.snowTest.testRunner.TestRunner">
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.snowTest.testRunner.TestRunner@4883b407]" name="setUpClass" is-config="true" duration-ms="1367" started-at="2019-08-04T22:17:12Z" finished-at="2019-08-04T22:17:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="PASS" signature="feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:com.snowTest.testRunner.TestRunner@4883b407]" name="feature" duration-ms="20103" started-at="2019-08-04T22:17:14Z" description="Run cucumber Features" data-provider="features" finished-at="2019-08-04T22:17:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Branch feature in Gurukula app]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- feature -->
        <test-method status="PASS" signature="feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:com.snowTest.testRunner.TestRunner@4883b407]" name="feature" duration-ms="39535" started-at="2019-08-04T22:17:34Z" description="Run cucumber Features" data-provider="features" finished-at="2019-08-04T22:18:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Login to Gurukula]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- feature -->
        <test-method status="FAIL" signature="feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:com.snowTest.testRunner.TestRunner@4883b407]" name="feature" duration-ms="11494" started-at="2019-08-04T22:18:13Z" description="Run cucumber Features" data-provider="features" finished-at="2019-08-04T22:18:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[As a new user I want to register a new Accoutn in Gurukula]]>
              </value>
            </param>
          </params>
          <exception class="cucumber.runtime.CucumberException">
            <message>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: expected [true] but found [false]
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at com.snowTest.testRunner.TestRunner.feature(TestRunner.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.snowTest.stepDefinitions.RegisterPageStepDefinitions.i_navigate_to_Registration_page(RegisterPageStepDefinitions.java:32)
	at âœ½.Then I navigate to Registration page "Registeration"(registerNewAccount.feature:12)
]]>
            </full-stacktrace>
          </exception> <!-- cucumber.runtime.CucumberException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- feature -->
        <test-method status="PASS" signature="testDownClass()[pri:0, instance:com.snowTest.testRunner.TestRunner@4883b407]" name="testDownClass" is-config="true" duration-ms="110" started-at="2019-08-04T22:18:25Z" finished-at="2019-08-04T22:18:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDownClass -->
      </class> <!-- com.snowTest.testRunner.TestRunner -->
    </test> <!-- Regression Test -->
  </suite> <!-- SNOW BDD Test Suite -->
</testng-results>
